tests/helpers.py

```diff
@@ -710,6 +723,24 @@ def get_cephfs_name():
     return result['items'][0].get('metadata').get('name')
 
 
+def pull_images(image_name):
+    """
+    Function to pull images on all nodes
+    Args:
+        image_name (str): Name of the container image to be pulled
+    Returns:
+        bool: True if deletion of image pull is successful
+    """
+
+    node_objs = node.get_node_objs(get_worker_nodes())
+    for node_obj in node_objs:
+        logging.info(f'pulling image "{image_name}" on node {node_obj.name}')
+        assert node_obj.ocp.exec_oc_debug_cmd(
+            node_obj.name, cmd_list=[f'podman pull {image_name}']
+        )
```

Comment:
I actually meant this way, node_objs = get_typed_node(nodetype="worker"), Never mind your implementation also looks good.